test-image.bats

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env bats
 
 @test "Test for node and version" {
-  run docker run --rm -it node:"$full_tag" node -e "process.stdout.write(process.versions.node)"
+  run docker run node:"$full_tag" node -e "process.stdout.write(process.versions.node)"
```

Comment:
I'm not sure how is this related with switching to GitHub Actions?

Reply:
I'm going to hold off changing anything for BATS because it sounds like https://github.com/nodejs/docker-node/pull/1197 might mean it gets dropped from the build

Reply:
No matter they are necessary or not, they're not related with switching to GitHub Actions, are they?

Reply:
From the OP

>Needed to remove the `-it` flag from the BATS testing since that doesn't seem to work on Actions

`--rm` should be kept regardless, annoying if running locally

Reply:
could keep the `-t` maybe? if it's only the interactive part which doesn't work

Reply:
If BATS gets removed in the other PR as redundant, i'll be removing the BATS install and the changes here

Reply:
I don't wanna remove bats unless somebody feels strongly about it. I think it provides a value by demonstrating node is runnable from the outside of the image build process.

Reply:
It's okay to keep it but the changes here should be separated into another PR.

Reply:
Added back the `--rm`